### ISSUE TITLE
[🐛 Bug]: Unable to hide DevTools listening on ws://127.0.0.1:53288/devtools/browser/5bc00fe8-5dea-4518-b602-70fae6c2378c

### ISSUE THREAD

**Issue by aamiralidev** [SeleniumHQ/selenium#12212]
### What happened?

I wrote a general automation script to take a few input parameters from users using PySide6 and then use Selenium to run the automation. Then to ship this, I used auto-py-to-exe with a simple 3 options path to the file, onedir, and windowed. After compilation, I ran the exe and it works fine except for some console windows that it shows and I don't want users to see those extra console windows. I tried to set the log level to CRITICAL, FATAL, and OFF through different os, logging, and Selenium web driver options. I successfully reduced logs to 1 line but this still shows the annoying console window that I want to hide. I'm attaching the screenshot of one such window, the other passes so quickly. 
![error](https://github.com/SeleniumHQ/selenium/assets/98078791/e3e7b798-8092-457c-9530-d5695328017f)

### How can we reproduce the issue?

```shell
Steps to reproduce issue
1. Create a sample PySide6 window
2. Hide it and create a selenium webdriver (Edge in this case)
3. run some random script or just put it to sleep
4. compile it using auto-py-to-exe
5. run it, you'll observer that a console window like one in screenshot shows before opening browser and it stays till the end of browser session
```

### Relevant log output

```shell
DevTools listening on ws://127.0.0.1:53288/devtools/browser/5bc00fe8-5dea-4518-b602-70fae6c2378c
```

### Operating System

Windows 11

### Selenium version

4.10.0

### What are the browser(s) and version(s) where you see this issue?

Microsoft Edge Version 114.0.1823.37 (Official build) (64-bit)

### What are the browser driver(s) and version(s) where you see this issue?

EdgeDriver 114.0.1823.37 (Official build) (64-bit)

### Are you using Selenium Grid?

No

**Comment by titusfortner** [comment]
We just merged #12103 which fixes this. By default, stderr/stdout for the whole chromedriver process should go to dev/null including that line.

**Comment by titusfortner** [comment]
Oh, and you also want to set the creation flag — https://github.com/SeleniumHQ/selenium/issues/12138#issuecomment-1574941188
